deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore/deployment.yaml

```diff
@@ -19,10 +19,10 @@ spec:
               mountPath: /data
           resources:
             limits:
-              cpu: "3"
+              cpu: "6"
               memory: 28Gi
```

Comment:
While at it bump the memory too now that dhstore runs on dedicated instance?

Reply:
makes sense - bumped